x-pack/docs/en/security/securing-communications/setting-up-ssl.asciidoc

```diff
@@ -9,8 +9,7 @@ WARNING: Clusters that do not have encryption enabled send all data in plain tex
 including passwords. If the {es} {security-features} are enabled, unless you have a trial license, you must configure SSL/TLS for internode-communication.
 
 The following steps describe how to enable encryption across the various
-components of the {stack}. You must perform each of the steps that are
-applicable to your cluster.
```

Comment:
not sure why we removed the last sentence.

Reply:
I don't really think this introduction is necessary anymore, so I've removed the whole thing